server/src/data-layer/services/BookingDataService.test.ts

```diff
@@ -0,0 +1,94 @@
+import { cleanFirestore } from "test-config/TestUtils"
+import BookingDataService from "./BookingDataService"
+import FirestoreCollections from "data-layer/adapters/FirestoreCollections"
+
+describe("BookingDataService", () => {
+  afterEach(async () => {
+    await cleanFirestore()
+  })
+
+  it("Should create a booking in firestore", async () => {
+    const { id } = await new BookingDataService().createBooking({
+      user_id: "asds",
+      booking_slot_id: "asdds",
+      stripe_payment_id: "stripe_payment_id_value"
+    })
+    const doc = await FirestoreCollections.bookings.doc(id).get()
+    const data = doc.data()
+
+    expect(data).not.toBe(undefined)
+    expect(data).toEqual({
+      user_id: "asds",
+      booking_slot_id: "asdds",
+      stripe_payment_id: "stripe_payment_id_value"
+    })
+  })
+
+  // it("Should get bookings based on specific id", async () => {
+  //   await new BookingDataService().createBooking({
+  //     user_id: "Person1",
+  //     booking_slot_id: "booking1",
+  //     stripe_payment_id: "stripeID1"
+  //   })
+
+  //   const bookingsBy_UID = await new BookingDataService().getBookingsByUserId("Person1") //test for user_id input
+
+  //   expect(bookingsBy_UID).not.toBe(undefined)
+  //   expect(bookingsBy_UID.length).toBe(1)
```

Comment:
If these are for separate functions it may be better to have a different test case for each